site/docs/education/tutorials/building-egeria-tutorial/terminal-installing-egeria.md

```diff
@@ -16,10 +16,10 @@ The name of the files is determined by the release level of the code that you [d
 ```bash
 antrun                                                  egeria-3.12-SNAPSHOT-distribution
 archive-tmp                                             egeria-3.12-SNAPSHOT-distribution.tar.gz
-egeria-3.12-SNAPSHOT-deploy                              open-metadata-assemblies-3.12-SNAPSHOT-javadoc.jar
-egeria-3.12-SNAPSHOT-deploy.tar.gz                       open-metadata-assemblies-3.12-SNAPSHOT-sources.jar
+egeria-3.12-SNAPSHOT-deploy                             open-metadata-assemblies-3.12-SNAPSHOT-javadoc.jar
+egeria-3.12-SNAPSHOT-deploy.tar.gz                      open-metadata-assemblies-3.12-SNAPSHOT-sources.jar
 ```
-The install image tar file is `{{release}}-distribution.tar.gz` or `egeria-3.12-SNAPSHOT-distribution.tar.gz` in this example.
+The distribution tar file is `{{release}}-distribution.tar.gz` or `egeria-3.12-SNAPSHOT-distribution.tar.gz` in this example.
```

Comment:
note that for the gradle build, the distribution is in a slightly different location to maven
```
./open-metadata-distribution/open-metadata-assemblies/build/   unpacked/egeria-${RELEASE}-distribution.tar.gz/server/${CHASSIS}-${RELEASE}.jar
```

Reply:
The whole purpose for updating this section is to change the location of the distribution and the instructions we have to assume a gradle build - the location shown is for the gradle build - not maven.